src/Attr.h

```diff
@@ -63,13 +67,11 @@ class Attr final : public Obj {
 
 	~Attr() override = default;
 
-	AttrTag Tag() const	{ return tag; }
+	AttrTag Tag() const { return tag; }
 
-	[[deprecated("Remove in v4.1.  Use GetExpr().")]]
-	Expr* AttrExpr() const	{ return expr.get(); }
+	[[deprecated("Remove in v4.1.  Use GetExpr().")]] Expr* AttrExpr() const { return expr.get(); }
```

Comment:
I really wish it wouldn't change lines like this and would just leave the attributes on lines by themselves. I don't think there's any options for it though.

Reply:
agree that thaat would be preferred, but seems the kind of things we'll just have to accept